test/powershell/Modules/Microsoft.PowerShell.Security/ExecutionPolicy.Tests.ps1

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 Import-Module HelpersCommon
```

Comment:
The same. Below I see more samples.
(Add new line after license. )

Reply:
```suggestion

Import-Module HelpersCommon
```